unitTests/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacetTestUtils.ts

```diff
@@ -19,7 +19,7 @@ export class DynamicHierarchicalFacetTestUtils {
       numberOfValues: 5,
       pageSize: 10,
       enableMoreLess: true,
-      basePath: [],
+      basePath: ['hello'],
```

Comment:
Do we need to handle splitting the string using the delimiter? Or will this be handled by the API?

If it will on the API side, should the option be a string rather than an array?

Reply:
I will add more values to display that fact. It is an array of strings